ポートフォリオの作成

KY1212 opened the pull request:
# 作成するパーツ
ポートフォリオのページ作成
実績ページの作成

# 対応するブランチ
feature/make-portfolio

daichan2019 commented:
ファイル差分おおすぎてchange logからレビューできません。
コメントでFBするので、以下をご確認ください。

1.
headerにclassを振りましょう。
FLOCSSを採用するなら適切なレイヤーのプレフィックスをつけて命名をしましょう。

2.グロナビのcontactがスペルミスです。
https://gyazo.com/72081d4613bb3e9f269c040a9fdeefee

3.header.php
`<div class="p-inner">`
どこのインナーですか。
FLOCSSでは、ブロックの下には必ずエレメントが発生します。
.p-inner__hoge
.p-inner__piyo
といったエレメントが子要素がこなければいけません。
headerをl-headerとし、
こちらの要素は機能をもったモジュールとしてのヘッダーという意味でプロジェクト層にきりだしp-headerとするか、
l-headerのエレメントと捉えl-header__innerと捉えるかのどちらかが最適です。

4.
sassファイルの記述方法の話をします。
object層にはprojectとcomponentが入ります。場合によってはutilityを含める人もいますね。
FLOCSSは「p-hoge」「c-piyo」などのモジュール（部品）ごとにファイルを分ける設計です。
ですので、`Object/_main.scss`にトップページのすべてのスタイルを記述するのは不適切です。
ましてやobjectのproject、componentどちらに該当するのかわかりにくいです。
ファイルの切り分けをしましょう。

5.
```css
header .p-inner {
	display: flex;
	justify-content: space-between;
	align-items: center;
}
```
flocssでは上記のようなスコープを切るような記述は原則しません。
スコープを切らずにBEMで単一classでスタイリングするのが強みですので見直しましょう。

6.
discription ➔ description
showmore ➔ show-more
firstview ➔ first-view
sumbnails ➔ thumbnails
です。
単語の区切りやタイポが目立ちます。
正しい単語の区切りにしてください。

7.Components/_components.scss
コンポーネントに「p-」は入りません。
こちらもコンポーネントごとにファイルの切り分けをしてください。


8.
素人目線でもうしわけないですが、ページのデザインは概ねいいんじゃないでしょうか！
ただ、Worksが増えていったときのことを考えるとテンプレートに切り出したほうがいいとは思います。
エスカレーションして突っ込まれなかったらOKということにしましょう。


9.
jsのfunction smoothScroll
こちらコピペ感満載のコードをpureなjsからjqueryの記述に修正しましょう。

10.
jsのvendorって使ってます？

KY1212 commented:
> ファイル差分おおすぎてchange logからレビューできません。
> コメントでFBするので、以下をご確認ください。
> 
> 
> headerにclassを振りましょう。
> FLOCSSを採用するなら適切なレイヤーのプレフィックスをつけて命名をしましょう。
> 
> 2.グロナビのcontactがスペルミスです。
> https://gyazo.com/72081d4613bb3e9f269c040a9fdeefee
> 
> 3.header.php
> `<div class="p-inner">`
> どこのインナーですか。
> FLOCSSでは、ブロックの下には必ずエレメントが発生します。
> .p-inner__hoge
> .p-inner__piyo
> といったエレメントが子要素がこなければいけません。
> headerをl-headerとし、
> こちらの要素は機能をもったモジュールとしてのヘッダーという意味でプロジェクト層にきりだしp-headerとするか、
> l-headerのエレメントと捉えl-header__innerと捉えるかのどちらかが最適です。
> 
> 
> sassファイルの記述方法の話をします。
> object層にはprojectとcomponentが入ります。場合によってはutilityを含める人もいますね。
> FLOCSSは「p-hoge」「c-piyo」などのモジュール（部品）ごとにファイルを分ける設計です。
> ですので、`Object/_main.scss`にトップページのすべてのスタイルを記述するのは不適切です。
> ましてやobjectのproject、componentどちらに該当するのかわかりにくいです。
> ファイルの切り分けをしましょう。
> 
> 
> ```css
> header .p-inner {
> 	display: flex;
> 	justify-content: space-between;
> 	align-items: center;
> }
> ```
> 
> flocssでは上記のようなスコープを切るような記述は原則しません。
> スコープを切らずにBEMで単一classでスタイリングするのが強みですので見直しましょう。
> 
> 
> discription ➔ description
> showmore ➔ show-more
> firstview ➔ first-view
> sumbnails ➔ thumbnails
> です。
> 単語の区切りやタイポが目立ちます。
> 正しい単語の区切りにしてください。
> 
> 7.Components/_components.scss
> コンポーネントに「p-」は入りません。
> こちらもコンポーネントごとにファイルの切り分けをしてください。
> 
> 
> 素人目線でもうしわけないですが、ページのデザインは概ねいいんじゃないでしょうか！
> ただ、Worksが増えていったときのことを考えるとテンプレート�に切り出したほうがいいとは思います。
> エスカレーションして突っ込まれなかったらOKということにしましょう。
> 
> 
> jsのfunction smoothScroll
> こちらコピペ感満載のコードをpureなjsからjqueryの記述に修正しましょう。
> 
> 
> jsのvendorって使ってます？



============================
==                       fixed                         ==
============================
ファイル差分おおすぎてchange logからレビューできません。 コメントでFBするので、以下をご確認ください。

1.headerにclassを振りましょう。 FLOCSSを採用するなら適切なレイヤーのプレフィックスをつけて命名をしましょう。

fixed
 .headerのタイポを修正, .innerの命名を修正
3826f0b9bb0d034609a7f5de7901b2f74ee874df


2.グロナビのcontactがスペルミスです。 https://gyazo.com/72081d4613bb3e9f269c040a9fdeefee

fixed
 .headerのタイポを修正, 3.innerの命名を修正
3826f0b9bb0d034609a7f5de7901b2f74ee874df

3.header.php <div class="p-inner"> どこのインナーですか。 FLOCSSでは、ブロックの下には必ずエレメントが発生します。 .p-inner__hoge .p-inner__piyo といったエレメントが子要素がこなければいけません。 headerをl-headerとし、 こちらの要素は機能をもったモジュールとしてのヘッダーという意味でプロジェクト層にきりだしp-headerとするか、 l-headerのエレメントと捉えl-header__innerと捉えるかのどちらかが最適です。

fixed
 .headerのタイポを修正, .innerの命名を修正
3826f0b9bb0d034609a7f5de7901b2f74ee874df


4.sassファイルの記述方法の話をします。 object層にはprojectとcomponentが入ります。場合によってはutilityを含める人もいますね。 FLOCSSは「p-hoge」「c-piyo」などのモジュール（部品）ごとにファイルを分ける設計です。 ですので、Object/_main.scssにトップページのすべてのスタイルを記述するのは不適切です。 ましてやobjectのproject、componentどちらに該当するのかわかりにくいです。 ファイルの切り分けをしましょう。

fixed FLOCSS思想に沿ったディレクトリ構成に修正
c7c6e49e0372bfd4e40565b995a43ddf8946f32f


5.	header .p-inner {
	  display: flex;
	  justify-content: space-between;
	  align-items: center;
        }
flocssでは上記のようなスコープを切るような記述は原則しません。 スコープを切らずにBEMで単一classでスタイリングするのが強みですので見直しましょう。

fixed コンポーネントのクラス名をFLOCSSの規則に沿った命名に修正
99c0d49574fa7f07d9a9430731c02e1f1fcf8fae


6.discription ➔ description showmore ➔ show-more firstview ➔ first-view sumbnails ➔ thumbnails です。 単語の区切りやタイポが目立ちます。 正しい単語の区切りにしてください。

fixed クラス名のタイポ修正
da642ee5a11e41fe034c4c5d64b4faa7129cd709


7.Components/_components.scss コンポーネントに「p-」は入りません。 こちらもコンポーネントごとにファイルの切り分けをしてください。

fixed コンポーネントのクラス名をFLOCSSの規則に沿った命名に修正
99c0d49574fa7f07d9a9430731c02e1f1fcf8fae



8.素人目線でもうしわけないですが、ページのデザインは概ねいいんじゃないでしょうか！ ただ、Worksが増えていったときのことを考えるとテンプレート�に切り出したほうがいいとは思います。 エスカレーションして突っ込まれなかったらOKということにしましょう。

※ここは突っ込まれたら修正する形にすることにします。



9.jsのfunction smoothScroll こちらコピペ感満載のコードをpureなjsからjqueryの記述に修正しましょう。

fixed Smooth Scrollの記述をpureJSからjQueryに修正しました
10cdc5e389a2f14c0a9f723944239a047b4bbf43


10.jsのvendorって使ってます？

fix functions.phpにてCDNからjQueryを読み込んでいたため、Vendorディレクトリを除去しました
633809881a6c1a88f8302e51cf68630128548941

KY1212 commented:
> > ファイル差分おおすぎてchange logからレビューできません。
> > コメントでFBするので、以下をご確認ください。
> > headerにclassを振りましょう。
> > FLOCSSを採用するなら適切なレイヤーのプレフィックスをつけて命名をしましょう。
> > 2.グロナビのcontactがスペルミスです。
> > https://gyazo.com/72081d4613bb3e9f269c040a9fdeefee
> > 3.header.php
> > `<div class="p-inner">`
> > どこのインナーですか。
> > FLOCSSでは、ブロックの下には必ずエレメントが発生します。
> > .p-inner__hoge
> > .p-inner__piyo
> > といったエレメントが子要素がこなければいけません。
> > headerをl-headerとし、
> > こちらの要素は機能をもったモジュールとしてのヘッダーという意味でプロジェクト層にきりだしp-headerとするか、
> > l-headerのエレメントと捉えl-header__innerと捉えるかのどちらかが最適です。
> > sassファイルの記述方法の話をします。
> > object層にはprojectとcomponentが入ります。場合によってはutilityを含める人もいますね。
> > FLOCSSは「p-hoge」「c-piyo」などのモジュール（部品）ごとにファイルを分ける設計です。
> > ですので、`Object/_main.scss`にトップページのすべてのスタイルを記述するのは不適切です。
> > ましてやobjectのproject、componentどちらに該当するのかわかりにくいです。
> > ファイルの切り分けをしましょう。
> > ```css
> > header .p-inner {
> > 	display: flex;
> > 	justify-content: space-between;
> > 	align-items: center;
> > }
> > ```
> > 
> > 
> >     
> >       
> >     
> > 
> >       
> >     
> > 
> >     
> >   
> > flocssでは上記のようなスコープを切るような記述は原則しません。
> > スコープを切らずにBEMで単一classでスタイリングするのが強みですので見直しましょう。
> > discription ➔ description
> > showmore ➔ show-more
> > firstview ➔ first-view
> > sumbnails ➔ thumbnails
> > です。
> > 単語の区切りやタイポが目立ちます。
> > 正しい単語の区切りにしてください。
> > 7.Components/_components.scss
> > コンポーネントに「p-」は入りません。
> > こちらもコンポーネントごとにファイルの切り分けをしてください。
> > 素人目線でもうしわけないですが、ページのデザインは概ねいいんじゃないでしょうか！
> > ただ、Worksが増えていったときのことを考えるとテンプレート�に切り出したほうがいいとは思います。
> > エスカレーションして突っ込まれなかったらOKということにしましょう。
> > jsのfunction smoothScroll
> > こちらコピペ感満載のコードをpureなjsからjqueryの記述に修正しましょう。
> > jsのvendorって使ってます？
> 
> # ============= fixed =============
> ファイル差分おおすぎてchange logからレビューできません。 コメントでFBするので、以下をご確認ください。
> 
> 1.headerにclassを振りましょう。 FLOCSSを採用するなら適切なレイヤーのプレフィックスをつけて命名をしましょう。
> 
> fixed
> .headerのタイポを修正, .innerの命名を修正
> [3826f0b](https://github.com/KY1212/ky_portfolio/commit/3826f0b9bb0d034609a7f5de7901b2f74ee874df)
> 
> 2.グロナビのcontactがスペルミスです。 https://gyazo.com/72081d4613bb3e9f269c040a9fdeefee
> 
> fixed
> .headerのタイポを修正, 3.innerの命名を修正
> [3826f0b](https://github.com/KY1212/ky_portfolio/commit/3826f0b9bb0d034609a7f5de7901b2f74ee874df)
> 
> 3.header.php 
> 
>  どこのインナーですか。 FLOCSSでは、ブロックの下には必ずエレメントが発生します。 .p-inner__hoge .p-inner__piyo といったエレメントが子要素がこなければいけません。 headerをl-headerとし、 こちらの要素は機能をもったモジュールとしてのヘッダーという意味でプロジェクト層にきりだしp-headerとするか、 l-headerのエレメントと捉えl-header__innerと捉えるかのどちらかが最適です。
> fixed
> .headerのタイポを修正, .innerの命名を修正
> [3826f0b](https://github.com/KY1212/ky_portfolio/commit/3826f0b9bb0d034609a7f5de7901b2f74ee874df)
> 
> 4.sassファイルの記述方法の話をします。 object層にはprojectとcomponentが入ります。場合によってはutilityを含める人もいますね。 FLOCSSは「p-hoge」「c-piyo」などのモジュール（部品）ごとにファイルを分ける設計です。 ですので、Object/_main.scssにトップページのすべてのスタイルを記述するのは不適切です。 ましてやobjectのproject、componentどちらに該当するのかわかりにくいです。 ファイルの切り分けをしましょう。
> 
> fixed FLOCSS思想に沿ったディレクトリ構成に修正
> [c7c6e49](https://github.com/KY1212/ky_portfolio/commit/c7c6e49e0372bfd4e40565b995a43ddf8946f32f)
> 
> 1. header .p-inner {
>    display: flex;
>    justify-content: space-between;
>    align-items: center;
>    }
>    flocssでは上記のようなスコープを切るような記述は原則しません。    スコープを切らずにBEMで単一classでスタイリングするのが強みですので見直しましょう。
> 
> fixed コンポーネントのクラス名をFLOCSSの規則に沿った命名に修正
> [99c0d49](https://github.com/KY1212/ky_portfolio/commit/99c0d49574fa7f07d9a9430731c02e1f1fcf8fae)
> 
> 6.discription ➔ description showmore ➔ show-more firstview ➔ first-view sumbnails ➔ thumbnails です。 単語の区切りやタイポが目立ちます。 正しい単語の区切りにしてください。
> 
> fixed クラス名のタイポ修正
> [da642ee](https://github.com/KY1212/ky_portfolio/commit/da642ee5a11e41fe034c4c5d64b4faa7129cd709)
> 
> 7.Components/_components.scss コンポーネントに「p-」は入りません。 こちらもコンポーネントごとにファイルの切り分けをしてください。
> 
> fixed コンポーネントのクラス名をFLOCSSの規則に沿った命名に修正
> [99c0d49](https://github.com/KY1212/ky_portfolio/commit/99c0d49574fa7f07d9a9430731c02e1f1fcf8fae)
> 
> 8.素人目線でもうしわけないですが、ページのデザインは概ねいいんじゃないでしょうか！ ただ、Worksが増えていったときのことを考えるとテンプレート�に切り出したほうがいいとは思います。 エスカレーションして突っ込まれなかったらOKということにしましょう。
> 
> ※ここは突っ込まれたら修正する形にすることにします。
> 
> 9.jsのfunction smoothScroll こちらコピペ感満載のコードをpureなjsからjqueryの記述に修正しましょう。
> 
> fixed Smooth Scrollの記述をpureJSからjQueryに修正しました
> [10cdc5e](https://github.com/KY1212/ky_portfolio/commit/10cdc5e389a2f14c0a9f723944239a047b4bbf43)
> 
> 10.jsのvendorって使ってます？
> 
> fix functions.phpにてCDNからjQueryを読み込んでいたため、Vendorディレクトリを除去しました
> [6338098](https://github.com/KY1212/ky_portfolio/commit/633809881a6c1a88f8302e51cf68630128548941)

KY1212 commented:
> > > ファイル差分おおすぎてchange logからレビューできません。
> > > コメントでFBするので、以下をご確認ください。
> > > headerにclassを振りましょう。
> > > FLOCSSを採用するなら適切なレイヤーのプレフィックスをつけて命名をしましょう。
> > > 2.グロナビのcontactがスペルミスです。
> > > https://gyazo.com/72081d4613bb3e9f269c040a9fdeefee
> > > 3.header.php
> > > `<div class="p-inner">`
> > > どこのインナーですか。
> > > FLOCSSでは、ブロックの下には必ずエレメントが発生します。
> > > .p-inner__hoge
> > > .p-inner__piyo
> > > といったエレメントが子要素がこなければいけません。
> > > headerをl-headerとし、
> > > こちらの要素は機能をもったモジュールとしてのヘッダーという意味でプロジェクト層にきりだしp-headerとするか、
> > > l-headerのエレメントと捉えl-header__innerと捉えるかのどちらかが最適です。
> > > sassファイルの記述方法の話をします。
> > > object層にはprojectとcomponentが入ります。場合によってはutilityを含める人もいますね。
> > > FLOCSSは「p-hoge」「c-piyo」などのモジュール（部品）ごとにファイルを分ける設計です。
> > > ですので、`Object/_main.scss`にトップページのすべてのスタイルを記述するのは不適切です。
> > > ましてやobjectのproject、componentどちらに該当するのかわかりにくいです。
> > > ファイルの切り分けをしましょう。
> > > ```css
> > > header .p-inner {
> > > 	display: flex;
> > > 	justify-content: space-between;
> > > 	align-items: center;
> > > }
> > > ```
> > > 
> > > 
> > >     
> > >       
> > >     
> > > 
> > >       
> > >     
> > > 
> > >     
> > >   
> > > flocssでは上記のようなスコープを切るような記述は原則しません。
> > > スコープを切らずにBEMで単一classでスタイリングするのが強みですので見直しましょう。
> > > discription ➔ description
> > > showmore ➔ show-more
> > > firstview ➔ first-view
> > > sumbnails ➔ thumbnails
> > > です。
> > > 単語の区切りやタイポが目立ちます。
> > > 正しい単語の区切りにしてください。
> > > 7.Components/_components.scss
> > > コンポーネントに「p-」は入りません。
> > > こちらもコンポーネントごとにファイルの切り分けをしてください。
> > > 素人目線でもうしわけないですが、ページのデザインは概ねいいんじゃないでしょうか！
> > > ただ、Worksが増えていったときのことを考えるとテンプレート�に切り出したほうがいいとは思います。
> > > エスカレーションして突っ込まれなかったらOKということにしましょう。
> > > jsのfunction smoothScroll
> > > こちらコピペ感満載のコードをpureなjsからjqueryの記述に修正しましょう。
> > > jsのvendorって使ってます？
> > 
> > 
> > # ============= fixed =============
> > ファイル差分おおすぎてchange logからレビューできません。 > コメントでFBするので、以下をご確認ください。
> > 1.headerにclassを振りましょう。 > FLOCSSを採用するなら適切なレイヤーのプレフィックスをつけて命名をしましょう。
> > fixed
> > .headerのタイポを修正, .innerの命名を修正
> > [3826f0b](https://github.com/KY1212/ky_portfolio/commit/3826f0b9bb0d034609a7f5de7901b2f74ee874df)
> > 2.グロナビのcontactがスペルミスです。 > https://gyazo.com/72081d4613bb3e9f269c040a9fdeefee
> > fixed
> > .headerのタイポを修正, 3.innerの命名を修正
> > [3826f0b](https://github.com/KY1212/ky_portfolio/commit/3826f0b9bb0d034609a7f5de7901b2f74ee874df)
> > 3.header.php > 
> >  > どこのインナーですか。 > FLOCSSでは、ブロックの下には必ずエレメントが発生します。 > .p-inner__hoge > .p-inner__piyo > といったエレメントが子要素がこなければいけません。 > headerをl-headerとし、 > こちらの要素は機能をもったモジュールとしてのヘッダーという意味でプロジェクト層にきりだしp-headerとするか、 > l-headerのエレメントと捉えl-header__innerと捉えるかのどちらかが最適です。
> > fixed
> > .headerのタイポを修正, .innerの命名を修正
> > [3826f0b](https://github.com/KY1212/ky_portfolio/commit/3826f0b9bb0d034609a7f5de7901b2f74ee874df)
> > 4.sassファイルの記述方法の話をします。 > object層にはprojectとcomponentが入ります。場合によってはutilityを含める人もいますね。 > FLOCSSは「p-hoge」「c-piyo」などのモジュール（部品）ごとにファイルを分ける設計です。 > ですので、Object/_main.scssにトップページのすべてのスタイルを記述するのは不適切です。 > ましてやobjectのproject、componentどちらに該当するのかわかりにくいです。 > ファイルの切り分けをしましょう。
> > fixed FLOCSS思想に沿ったディレクトリ構成に修正
> > [c7c6e49](https://github.com/KY1212/ky_portfolio/commit/c7c6e49e0372bfd4e40565b995a43ddf8946f32f)
> > 
> > 1. header .p-inner {
> >    display: flex;
> >    justify-content: space-between;
> >    align-items: center;
> >    }
> >    flocssでは上記のようなスコープを切るような記述は原則しません。 >       スコープを切らずにBEMで単一classでスタイリングするのが強みですので見直しましょう。
> > 
> > fixed コンポーネントのクラス名をFLOCSSの規則に沿った命名に修正
> > [99c0d49](https://github.com/KY1212/ky_portfolio/commit/99c0d49574fa7f07d9a9430731c02e1f1fcf8fae)
> > 6.discription ➔ description > showmore ➔ show-more > firstview ➔ first-view > sumbnails ➔ thumbnails > です。 > 単語の区切りやタイポが目立ちます。 > 正しい単語の区切りにしてください。
> > fixed クラス名のタイポ修正
> > [da642ee](https://github.com/KY1212/ky_portfolio/commit/da642ee5a11e41fe034c4c5d64b4faa7129cd709)
> > 7.Components/_components.scss > コンポーネントに「p-」は入りません。 > こちらもコンポーネントごとにファイルの切り分けをしてください。
> > fixed コンポーネントのクラス名をFLOCSSの規則に沿った命名に修正
> > [99c0d49](https://github.com/KY1212/ky_portfolio/commit/99c0d49574fa7f07d9a9430731c02e1f1fcf8fae)
> > 8.素人目線でもうしわけないですが、ページのデザインは概ねいいんじゃないでしょうか！ > ただ、Worksが増えていったときのことを考えるとテンプレート�に切り出したほうがいいとは思います。 > エスカレーションして突っ込まれなかったらOKということにしましょう。
> > ※ここは突っ込まれたら修正する形にすることにします。
> > 9.jsのfunction smoothScroll > こちらコピペ感満載のコードをpureなjsからjqueryの記述に修正しましょう。
> > fixed Smooth Scrollの記述をpureJSからjQueryに修正しました
> > [10cdc5e](https://github.com/KY1212/ky_portfolio/commit/10cdc5e389a2f14c0a9f723944239a047b4bbf43)
> > 10.jsのvendorって使ってます？
> > fix functions.phpにてCDNからjQueryを読み込んでいたため、Vendorディレクトリを除去しました
> > [6338098](https://github.com/KY1212/ky_portfolio/commit/633809881a6c1a88f8302e51cf68630128548941)


Fixed innerの命名を修正、Componentの中身を個々のファイルに切り分け、Contact-form7.scssをUtilityに移動しました
e0a6f76664ee03a9f92faeb269948b44a587458d

daichan2019 commented:
FBします。

1.
いらないHTMLファイルは削除しましょう。

2.
https://gyazo.com/6cf3cd912fbed41cf9319a4a60fcd58e
.p-profile__details
この要素でmargin-leftを取らず、p-profile__profile-wrapで取れます。
親で取れる余白は親で取りましょう。

3.
span.scss
```css
//管理画面の投稿画面にあるBの部分で付与するスタイル
//strongではなくspanに変更
span.my_bold {
	font-weight: 700;
}

```
これ、`u-font-bold`とかで解決できないですか？？
baseに記載するでもおｋです。

4.
https://gyazo.com/0915044d176dd09122aad869f0f11bd8
このpadding-topってなぜここでとったのですか？


5.
p-single__deliverables
これ「成果物」って命名にしたかったら「p-work」とかじゃ駄目ですか？
シンプルに長いです。
あとこれp-singleのエレメントにしてますが、中にエレメント結構多く詰まっていますので、
ブロックに切り出したほうがいいと思うのですがいかがでしょうか？？


6.
https://gyazo.com/3ad4cc182b3d85eab02efa2d08461daa
ここのaタグ、スタイル当ててるならclassを振りましょう


7.
コメントの残し方、見づらいので一つ一つ引用する形で残してください！

↓例

> 10. jsのvendorって使ってます？
こちら使っていませんので、削除いたしました！

KY1212 commented:
**1. いらないHTMLファイルは削除しましょう。**

**fixed** 不要なHTMLファイルを削除しました。
b9a4284
<br> 
**2. この要素でmargin-leftを取らず、p-profile__profile-wrapで取れます。
親で取れる余白は親で取りましょう。**

**fixed** profileセクション内の不要クラス(p-profile__profile-wrap)を削除しました。
ffdf0ed
<br> 
**3. '''span.my_bold { font-weight: 700; }```
これ、u-font-boldとかで解決できないですか？？**

**fixed** Utility/spanに付与しているクラス名を変更しました。font-weight:boldとなっている箇所を700に統一しました。
b03ec95
<br> 
**4. このpadding-topってなぜここでとったのですか？**

**fixed** Singleページにて、.p-single__deliverablesにあたっていた不要なpaddingを除去しました
c4f4d5c
 <br> 
**5. これ「成果物」って命名にしたかったら「p-work」とかじゃ駄目ですか？
あとこれp-singleのエレメントにしてますが、中にエレメント結構多く詰まっていますので、
ブロックに切り出したほうがいいと思うのですがいかがでしょうか？？**

**fixed** Singleページにて、大枠のクラスの命名を変更、それに伴い、配下のクラスの命名も変更しました
d7d2f73
<br>  
**6. works配下ボタン部分のaタグにスタイル当てているのであれば、class名を振りましょう**

**fixed** worksセクションにて、ボタンのaタグにクラス名を振り、親にc-btnを付与しました
b9c0e84

KY1212 commented:
> **1. いらないHTMLファイルは削除しましょう。**
> 
> **fixed** 不要なHTMLファイルを削除しました。
> [b9a4284](https://github.com/KY1212/ky_portfolio/commit/b9a42844986bbdc1a82b22f3fd6b57fba18163bd)
> 
> 
> **2. この要素でmargin-leftを取らず、p-profile__profile-wrapで取れます。 親で取れる余白は親で取りましょう。**
> 
> **fixed** profileセクション内の不要クラス(p-profile__profile-wrap)を削除しました。
> [ffdf0ed](https://github.com/KY1212/ky_portfolio/commit/ffdf0ed6d2cf3c79358bbc4a309fd1e9333929eb)
> 
> 
> **3. '''span.my_bold { font-weight: 700; }``` これ、u-font-boldとかで解決できないですか？？**
> 
> **fixed** Utility/spanに付与しているクラス名を変更しました。font-weight:boldとなっている箇所を700に統一しました。
> [b03ec95](https://github.com/KY1212/ky_portfolio/commit/b03ec9517832f624d74430cd7a2bc9bd8c7643f2)
> 
> 
> **4. このpadding-topってなぜここでとったのですか？**
> 
> **fixed** Singleページにて、.p-single__deliverablesにあたっていた不要なpaddingを除去しました
> [c4f4d5c](https://github.com/KY1212/ky_portfolio/commit/c4f4d5c28ebedf10419caaffd77d790981b7319f)
> 
> 
> **5. これ「成果物」って命名にしたかったら「p-work」とかじゃ駄目ですか？ あとこれp-singleのエレメントにしてますが、中にエレメント結構多く詰まっていますので、 ブロックに切り出したほうがいいと思うのですがいかがでしょうか？？**
> 
> **fixed** Singleページにて、大枠のクラスの命名を変更、それに伴い、配下のクラスの命名も変更しました
> [d7d2f73](https://github.com/KY1212/ky_portfolio/commit/d7d2f73eead9b5b21910618719eaa9a7451ffe03)
> 
> 
> **6. works配下ボタン部分のaタグにスタイル当てているのであれば、class名を振りましょう**
> 
> **fixed** worksセクションにて、ボタンのaタグにクラス名を振り、親にc-btnを付与しました
> [b9c0e84](https://github.com/KY1212/ky_portfolio/commit/b9c0e84c38229dc1403f8214a7eb02e93c9ccef5)

daichan2019 commented:
FBします。

1.
```css
.u-font-weight__bold {
	font-weight: 700;
}
```
命名NGです。
基本utilityはblockの中にelementが存在する構造とは別のレイヤーですので、block__elementの形にはならないです。
やるとしたら、`.u-font-bold`とかです。

2.
```css
.c-btn__double-line
```
この命名も1.同様`c-`はblock__elementの構造を取りませんので修正しましょう。
というか、`c-btn`のmodifierと捉えることできませんか？
c-btnに関与する記述は一つのファイルを見ればわかるようにしたほうがいいです。

3.
全体的にPHPファイルのインデント揃えてください。
vscodeにインデント統一できるプラグインあるので調べて導入しましょう。


4.
.p-first-title__title
そもそもp-first-titleというblockが親要素に存在していません。
親にblockが存在しないのにいきなりblock__elementが出てくることはFLOCSSではありえません。


5.
```css
.p-first-view::before {
	background: url(../img/tropical.png) no-repeat center center/cover;
	background-attachment: fixed;
	content: "";
	display: block;
	height: 100%;
	left: 0;
	position: absolute;
	top: 0;
	width: 100%;
	opacity: 0.3;
}
```
これ、
```css
.p-first-view {

  &::before {}
}
```
でかける。

6.
```css
.p-header__item + .p-header__item {
	margin-left: 20px;
}
```
これも
```css
.p-header__item {
 & + & {}
}
```
でかける。

7.
//========works========
scssファイルに散見されるコメントアウトいらないので削除すること。

8.
ファイルの命名について
`l-` `c-`はファイル名にもプレフィックスをつけているのにprojectにはつけていないのでどちらかに統一してください。

9.
Foundationディレクトリと同階層にあるComponentディレクトリは不要です。削除しましょう。

KY1212 commented:
1.blockの中にelementが存在する構造とは別のレイヤーですので、block__elementの形にはならないです。 
Utilityの立ち位置を考えて命名してください。

fixed Utilityの命名をFLOCSSの規則に沿った命名に変更しました
dc75b78c93ba3f5ac36b843abaf221bcf75a970b

2. `.c-btn__double-line`
この命名も1.同様c-はblock__elementの構造を取りませんので修正しましょう。
というか、c-btnのmodifierと捉えることできませんか？
c-btnに関与する記述は一つのファイルを見ればわかるようにしたほうがいいです。

fixed c-btnの命名をFLOCSSに沿った命名規則に変更しました。
909421c73e7ad227784eab1debc90ae2716ebcc4
※このボタンは統一部分がなく、それぞれ別個のボタンとして設置しているのでModifierでやらずファイルは各々分けています。


3.全体的にPHPファイルのインデント揃えてください。
vscodeにインデント統一できるプラグインあるので調べて導入しましょう。

4.`.p-first-title__title`
そもそもp-first-titleというblockが親要素に存在していません。
親にblockが存在しないのにいきなりblock__elementが出てくることはFLOCSSではありえません。

fixed 命名がおかしい部分first-title__titleをfirst-view__titleに修正しました。
370af93ee406f9c4ea941c7902a4d01462819b86

5.擬似要素等付与している場合、&(アンパサンド)でかけること。

fixed ＆で囲える部分は囲むように修正しました。
6dc492298146402f91c1e6f3f4853c727fbb1878

6.


7.


8.

daichan2019 commented:
FBします！

１．
.p-first-view__title
これを`h1`タグ使ってほしいです！
```html
<h1>YUSUKE KATO'S</h1>
<h1>PORTFOLIO</h1>
```
これはspanでOKです


２．
ページ内に`h1`点在しすぎじゃないですか？
Googleの解析的には特に問題ないことが証明されているようですが、
ページ構造的にhタグの優先度を適切につけましょう

３．
sass/Components
このディレクトリを削除してって指示しませんでしたっけ？

KY1212 commented:
FBします！

１．
.p-first-view__title
これを`h1`タグ使ってほしいです！
```html
<h1>YUSUKE KATO'S</h1>
<h1>PORTFOLIO</h1>
```
これはspanでOKです

fixed divで囲っていた要素（p-first-view__title）をh1に変更しました。
813ca0145d68450ad35b62e9feeb6a7265b9f9f0
<br>
２．
ページ内に`h1`点在しすぎじゃないですか？
Googleの解析的には特に問題ないことが証明されているようですが、
ページ構造的にhタグの優先度を適切につけましょう

３．
sass/Components
このディレクトリを削除してって指示しませんでしたっけ？

daichan2019 commented:
FBします！

p-profile__descliption
これスペルミスです


上記修正しておいてください！
飯村さんへFB依頼出します！

daichan2019 commented:
8/3 FB

1.
https://gyazo.com/ab84148a22365e263fb72d676be61bde
これって、c-heading以外のh2はすべてこのスタイルが適用されるわけですよね？
保守性の観点でNGかと思うので修正しましょう。

2.
https://gyazo.com/01b0200aadf9eec85bfdcc8916018867
`p`と`a`が逆かと思うのですがいかがでしょうか？

3.
これ、使ってないので、消しましょう。
https://gyazo.com/01378ed46c2d2ca72c8c3761387d9e0c

4.
.p-works__post-wrap
のmarginが.p-works__innerを突き破ってます。

5.
```css
.p-single__image-thumbnail {
	padding: calc(172 / 300 * 100%) 20px 0;
	background-position: center center;
	background-size: cover;
}
```
paddingの値がマジックナンバーなので、変数化して適切な命名をした方が第三者目線で助かるかもです。

6.
work.scss
```scss
.p-works__post {
	color: $color-works-post;
	background-color: $bg-color-works-post;
	overflow: hidden;
	position: relative;
	width: calc(50% - 15px);
	@include mq(sp) {
		width: 100%;
	}
	a {
		display: block;
		padding: 20px;
		.p-works__post-thumbnail {
			padding: calc(172 / 300 * 100%) 20px 0;
			background-position: center center;
			background-size: cover;
			margin-top: 20px;
		}
	}
}
```
FLOCSSではカスケーディングは許容されていません。

7.
```css
.p-works__btn {
	line-height: 1;
	padding: 10px 30px;
	border: 10px double #8e1a47;
	color: #8e1a47;
	background-color: #f2ecde;
	transition-duration: 0.2s;
	&:hover {
		border: 10px double #f2ecde;
		color: #f2ecde;
		background-color: #8e1a47;
	}
}
```
colorの指定は変数に格納してしようするようにしましょう。

KY1212 commented:
8/3 FB

https://gyazo.com/ab84148a22365e263fb72d676be61bde
これって、c-heading以外のh2はすべてこのスタイルが適用されるわけですよね？
保守性の観点でNGかと思うので修正しましょう。

**Fixed**
singleページにて、h2に付与するスタイルをカスケーディングせず、別途クラスを用意しました
b6e817a370d424bb01ce0cbced11ac1d2436e383


https://gyazo.com/01b0200aadf9eec85bfdcc8916018867
pとaが逆かと思うのですがいかがでしょうか？

**Fixed**

これ、使ってないので、消しましょう。
https://gyazo.com/01378ed46c2d2ca72c8c3761387d9e0c

**Fixed**

.p-works__post-wrap
のmarginが.p-works__innerを突き破ってます。

**Fixed**

.p-single__image-thumbnail {
	padding: calc(172 / 300 * 100%) 20px 0;
	background-position: center center;
	background-size: cover;
}
paddingの値がマジックナンバーなので、変数化して適切な命名をした方が第三者目線で助かるかもです。

**Fixed**

work.scss

.p-works__post {
	color: $color-works-post;
	background-color: $bg-color-works-post;
	overflow: hidden;
	position: relative;
	width: calc(50% - 15px);
	@include mq(sp) {
		width: 100%;
	}
	a {
		display: block;
		padding: 20px;
		.p-works__post-thumbnail {
			padding: calc(172 / 300 * 100%) 20px 0;
			background-position: center center;
			background-size: cover;
			margin-top: 20px;
		}
	}
}
FLOCSSではカスケーディングは許容されていません。

.p-works__btn {
	line-height: 1;
	padding: 10px 30px;
	border: 10px double #8e1a47;
	color: #8e1a47;
	background-color: #f2ecde;
	transition-duration: 0.2s;
	&:hover {
		border: 10px double #f2ecde;
		color: #f2ecde;
		background-color: #8e1a47;
	}
}
colorの指定は変数に格納してしようするようにしましょう。

daichan2019 commented:
8/17 FBします

1.
https://gyazo.com/d1828072481af9fb5f566dab9cc4a136
p-single__item
こちらsingleの中のheadingなので命名修正したほうがいいかと。
ex) `.p-single__item` ➔ `.p-single__headline`

2.
base.scss
```css
h2 {
	& + & {
		margin-top: 30px;
	}
	&:before {
		color: $color-h2;
		font-family: "Font Awesome 5 Free";
		content: "\f138";
		font-weight: 900;
		position: absolute;
		font-size: 1em;
		left: 0.25em;
		top: 0.5em;
	}
}
```
これってh2にこのスタイルがすべて適用されるわけですよね？
保守性の観点でNGかと思います。
今の規模感ではベースのCSSに設定するほうが楽かと思ってこのような記述にしているのかと思いますが、ポートフォリオって後に改修する頻度結構高いし、デザインのテコ入れとかも発生しそうですので、classを設けて管理したほうがベターかと。

3.
var.scss
```css
$color-header: #6a1435;
$color-profile: #fff;
$color-works: #333;
$color-works-title: #333;
$color-works-post: #fff;
$color-contact: #333;
$color-footer: #fff;
```
せっかく場所ごとにカラー変数設定しているのであれば以下の書き方のほうが良さそう

```css
$color-white: #fff !default;
$color-dark: #333 !default;

$color-contact: $color-dark !default;
$color-footer: $color-white !default;
```
サイト全体で使うカラーを変数に格納して、
場所ごとにその変数を利用する。

この方が変更箇所少なくてすむ。
あと末尾に`!default`つけましょう

4.
```css
.l-contact {
	background-color: $bg-color-contact;
}

.l-single {
	padding: 50px 0;
	flex: 1;
}
```
こちら使ってます？
適用していないスタイルのファイルは削除してください

5.
```css
h1 {
	font-weight: 700;
}

h3 {
	font-weight: 700;
}
```
こちらもhタグ使用した時点でfont-weightかかりますよね。
class名管理でなおかつprojectで当てに行っていいです

6.
```css
.p-single__item {
	position: relative;
	padding: 0.5em 0.5em 0.5em 1.5em;
	line-height: 1.4;
	color: $single-item-color;
	border-top: dotted 1px $single-item-border;
	border-bottom: dotted 1px $single-item-border;
	background: $single-item-bg;
	margin-bottom: 10px;
	font-weight: 700;
	letter-spacing: 2px;
}
```
ところどころ
`em`使用していますが理由教えていただけますか？

7.
細かいところだけれども
https://gyazo.com/90484a0f612104c334247de25944e6a9
ディレクトリ名は
単数形 or 複数形どちらかで統一する
